WcaOnRails/spec/models/team_spec.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Team do
+  before(:all) do
+    # Names were generated from FactoryBot.create :user
+    @names = ["Larry Ullrich", "Jeffry Kiehn", "Colby Nader", "Lakia Cremin", "Shanae Price", "Ka Hermiston", "Milton Pfeffer", "Jolie Cartwright", "Alana Monahan", "Elden Cruickshank"]
```

Comment:
Not necessary. Just create N users (where N is any number you require for your tests) by invoking something like
```
let!(:dummy_users) { FactoryBot.create_list :user, N }
```

Reply:
I'm actually giving the list sorted to WEAT, because they are supposed to show the members in alphabetically sorted. So, is it fine if I hard code names? Or is there any other solution?